ledger/evalbench_test.go

```diff
@@ -239,10 +287,58 @@ func BenchmarkBlockEvaluatorDiskNoCrypto(b *testing.B) {
 }
 
 func BenchmarkBlockEvaluatorDiskAppOptIns(b *testing.B) {
-	g := BenchAppOptInsTxnGenerator{
-		NumApps: 500,
-		Proto:   protocol.ConsensusFuture,
-		Program: []byte{0x02, 0x20, 0x01, 0x01, 0x22},
+	g := benchAppOptInsTxnGenerator{
+		NumApps:          500,
+		Proto:            protocol.ConsensusFuture,
+		Program:          []byte{0x02, 0x20, 0x01, 0x01, 0x22},
+		MaxAppsOptedIn:   config.Consensus[protocol.ConsensusV30].MaxAppsOptedIn,
+		TransactionsType: protocol.PaymentTx,
+	}
+	benchmarkBlockEvaluator(b, false, false, protocol.ConsensusFuture, &g)
+}
+
+func BenchmarkBlockEvaluatorDiskAppCalls(b *testing.B) {
+	if b.N < 2 {
+		b.N = 2
+	}
```

Comment:
Why is this needed?

Reply:
Please add a comment about this.
Moreover, `benchmarkBlockEvaluator` uses b.N only in
```
	for i := 0; i < b.N; i++ {
		if withCrypto {
			_, err = l2.Validate(context.Background(), validatedBlock.Block(), backlogPool)
		} else {
			_, err = internal.Eval(context.Background(), l2, validatedBlock.Block(), false, nil, nil)
		}
		require.NoError(b, err)
	}
```
with b.N in range [1,2] does not make much sense.